### PR TITLE
asserts,client: switch snap-build and snap-revision to be indexed by snap-sha3-384

### PR DESCRIPTION
**Pull request by pedronis**
This also drop series and snap-digest from them.

### CODE REVIEW

**Comment by pedronis** [comment]
retest this please

File: asserts/snap_asserts_test.go [hunk 1]
```diff
@@ -228,25 +230,25 @@ const (
 )
 
 func (sbs *snapBuildSuite) TestDecodeInvalid(c *C) {
+	hh := "snap-sha3-384: " + blobSHA3_384 + "\n"
```
Comment (emgee) [inline asserts/snap_asserts_test.go]: maybe "hashh" for a tiny bit more meaning? ;)

File: asserts/snap_asserts_test.go [hunk 2]
```diff
@@ -405,13 +402,15 @@ const (
 
 func (srs *snapRevSuite) TestDecodeInvalid(c *C) {
 	encoded := srs.makeValidEncoded()
+
+	hh := "snap-sha3-384: " + blobSHA3_384 + "\n"
```
Comment (emgee) [inline asserts/snap_asserts_test.go]: maybe "hashh"?

**Comment by emgee** [comment]
+1

fwiw, I didn't like the release.Series in SnapRevision.checkConsistency but it's good enough for now and some sort of "context" to make the series configurable can be added later.

**Comment by chipaca** [comment]
:+1:

**Pull request merged**